src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -46,8 +46,6 @@ public static class Args
                     {
                         Password,
                         Token,
```

Comment:
For organizational consistency, it would be good to move the fields as well (a few lines up) out of the grouping marked as "secret args".